internal/beater/api/config/agent/handler_test.go

```diff
@@ -233,9 +208,9 @@ func TestAgentConfigHandlerAnonymousAccess(t *testing.T) {
 }
 
 func TestAgentConfigHandlerAuthorizedForService(t *testing.T) {
-	cfg := config.KibanaAgentConfig{Cache: config.Cache{Expiration: time.Nanosecond}}
-	f := agentcfg.NewKibanaFetcher(nil, cfg.Cache.Expiration)
-	h := NewHandler(f, cfg, "", nil)
+	// TODO
```

Comment:
Is there anything left todo in this test?

Reply:
Oops, thanks. I intended to replace the `&kibana.Client{}` with an actual client. Fixed.